src/components/ItemFilters/ItemFilters.test.tsx

```diff
@@ -8,75 +8,101 @@ import Bib from "../../models/Bib"
 
 describe("ItemFilters", () => {
   let filtersChangeMock: jest.Mock
-  beforeEach(() => {
-    const bib = new Bib(bibWithItems.resource)
-    filtersChangeMock = jest.fn()
-    render(
-      <ItemFilters
-        itemAggregations={bib.itemAggregations}
-        handleFiltersChange={filtersChangeMock}
-        appliedFilters={{
-          location: ["Offsite"],
-          format: ["Text"],
-          status: ["Available"],
-          year: [],
-        }}
-        filtersAreApplied={true}
-      />
-    )
-  })
 
-  it("renders the Filter bar container", () => {
-    expect(screen.getByTestId("item-filters-container")).toBeInTheDocument()
-  })
+  describe("ItemFilters with year filter", () => {
+    let filtersChangeMock: jest.Mock
+    beforeEach(() => {
+      const bib = new Bib(bibWithItems.resource)
+      filtersChangeMock = jest.fn()
+      render(
+        <ItemFilters
+          itemAggregations={bib.itemAggregations}
+          handleFiltersChange={filtersChangeMock}
+          appliedFilters={{
+            location: ["Offsite"],
+            format: ["Text"],
+            status: ["Available"],
+            year: [],
+          }}
+          filtersAreApplied={true}
+          showDateFilter={true}
+        />
+      )
+    })
 
-  it("renders filters container with one MultiSelect per populated checkbox group", async () => {
-    expect(screen.getByTestId("item-filters-container")).toBeInTheDocument()
-    expect(screen.getByTestId("format-multi-select")).toBeInTheDocument()
-    expect(screen.getByTestId("status-multi-select")).toBeInTheDocument()
-  })
+    it("renders the Filter bar container", () => {
+      expect(screen.getByTestId("item-filters-container")).toBeInTheDocument()
+    })
 
-  it("renders the year search form", async () => {
-    expect(screen.getByLabelText("Search by year")).toBeInTheDocument()
-  })
+    it("renders filters container with one MultiSelect per populated checkbox group", async () => {
+      expect(screen.getByTestId("item-filters-container")).toBeInTheDocument()
+      expect(screen.getByTestId("format-multi-select")).toBeInTheDocument()
+      expect(screen.getByTestId("status-multi-select")).toBeInTheDocument()
+    })
 
-  it("closes open filters when user clicks outside of the filter", async () => {
-    const outsideOfTheFilter = screen.getByLabelText("Search by year")
+    it("renders the year search form when showDateFilter is true", async () => {
+      expect(screen.getByLabelText("Search by year")).toBeInTheDocument()
+    })
 
-    const locationFilterButton = screen
-      .getByTestId("location-multi-select")
-      .querySelector("button")
-    await userEvent.click(locationFilterButton)
+    it("closes open filters when user clicks outside of the filter", async () => {
```

Comment:
This is fine to keep here but also note that this is an implementation detail of the component, no? And this test already exists in the DS (in a similar way) so there's no need to replicate the test here, but okay to keep if you want to.

Reply:
Correct, this is a holdover from when this functionality was in research-catalog. This diff indicates that I moved it, but can remove it as part of this PR since it doesn't need to test the component's implementation.

Reply:
Gotcha. That wasn't clear. So then these tests were added back when this was not a default feature of the DS and was app-specific. I think they can be removed since the feature and tests are in the DS component.